gen/src/MPIgenerator.jl

```diff
@@ -19,6 +19,13 @@ module MPIgenerator
 
         options = load_options(joinpath(@__DIR__, "generator.toml"))  # wrapper generator options
 
+        doc_url = if MPIPreferences.binary == "MPICH_jll"
+            fname -> "- `$fname` man page `https://www.mpich.org/static/docs/latest/www3/$fname.html`"
+        else
+            fname -> "- `$fname` man page `https://www.open-mpi.org/doc/current/man3/$fname.3.php`"
+        end
+        options["general"]["callback_documentation"] = node -> String[doc_url(node.id)]
```

Comment:
why not link to both?

Reply:
Also, you can use  https://github.com/JuliaParallel/MPI.jl/blob/4e8dcd03fb428ee37abb963052cd6009a9aed45b/src/MPI.jl#L30-L34

Reply:
Done, both `OpenMPI` and `MPICH`.